src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -69,7 +69,17 @@ Copyright (c) .NET Foundation. All rights reserved.
     <AssemblySearchPaths>{CandidateAssemblyFiles};{HintPathFromItem};{TargetFrameworkDirectory};{RawFileName}</AssemblySearchPaths>
     <DesignTimeAssemblySearchPaths>$(AssemblySearchPaths)</DesignTimeAssemblySearchPaths>
   </PropertyGroup>
+  
+  <!-- Default glob settings -->
+  <PropertyGroup>
+    <GlobalExclude>$(GlobalExclude);bin\**;obj\**;packages\**</GlobalExclude>
```

Comment:
I was wrong, it should be here. But if it's here, no need to follow the "augment the previous definition" pattern since there won't be one.